addons/colorblind/addon.json

```diff
@@ -7,12 +7,6 @@
       "link": "https://github.com/DNin01"
     }
   ],
-  "info": [
-    {
-      "id": "feedback",
-      "text": "We want to expand this addon with other accessibility features in the future. You can send us your feedback by going to scratchaddons.com/feedback or clicking \"Send Feedback\" on the sidebar."
```

Comment:
Yeah, I'm pretty happy with how the addon is doing as just a link style customization. I could see other color blindness accommodations finding their home in new addons.